WordPress/Classes/ViewRelated/Stats/Period Stats/Countries/CountriesCell.swift

```diff
@@ -58,6 +62,13 @@ private extension CountriesCell {
     }
 
     func setSubtitleVisibility() {
+
+        guard !forDetails else {
```

Comment:
I think avoid the double negative here is more clear to read.
```
if forDetails {
.....
return
}
```
What do you think?

Reply:
Sure. It does read better. I've changed that one. And since I had done the exact same thing in `TopTotalsCell`, I went ahead & changed that one too.